Task_DEV_5/Task_DEV_5/FirstSorter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class FirstTextSorter
+    {
+        public string path;
```

Comment:
For fields more preferable private modificator. For properties - public modificator.

Reply:
Done.